app/tests/helpers/sharedTests.ts

```diff
@@ -0,0 +1,59 @@
+import { act, render, screen } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import mockRouter from 'next-router-mock'
+import singletonRouter from 'next/router'
+import { ReactElement } from 'react'
+import renderer from 'react-test-renderer'
+
+import { UserEventReturn } from './setup'
+
+export function testSnapshot(element: ReactElement) {
+  const tree = renderer.create(element).toJSON()
+  expect(tree).toMatchSnapshot()
```

Comment:
I likely missed the context somewhere, but why are y'all taking JSON snapshots of React elements as opposed to `expect(element).toMatchSnapshot()`? Maybe worth a code comment.

Reply:
@sawyerh Ah! Thank you for the example. I have been doing it this way because it is the example on the [jest snapshot docs](https://jestjs.io/docs/snapshot-testing), but your way seems much better.